ci/official/any.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+source "${BASH_SOURCE%/*}/utilities/setup.sh"
+
+# Parse options and build targets into arrays, so that shelllint doesn't yell
+# about readability. We can't pipe into 'read -ra' to create an array because
+# piped commands run in subshells, which can't store variables outside of the
+# subshell environment.
+# Ignore grep failures since we're using it for basic filtering
+set +e
+filtered_build_targets=( $(echo "$BUILD_TARGETS" | tr ' ' '\n' | grep .) )
+nonpip_targets=( $(echo "$TEST_TARGETS" | tr ' ' '\n' | grep -E "^//tensorflow/" ) )
+config=( $(echo "$CONFIG_OPTIONS" ) )
+test_flags=( $(echo "$TEST_FLAGS" ) )
+set -e
+
+if [[ "$TFCI_NVIDIA_SMI_ENABLE" == 1 ]]; then
+  tfrun nvidia-smi
+fi
+
+if [[ "${#filtered_build_targets[@]}" -ne 0 ]]; then
+  tfrun bazel "${TFCI_BAZEL_BAZELRC_ARGS[@]}" "${config[@]}" "${filtered_build_targets[@]}"
+fi
+
+if [[ "${PIP_WHEEL}" -eq "1" ]]; then
+  # Update the version numbers to build a "nightly" package
+  [[ "$TFCI_NIGHTLY_UPDATE_VERSION_ENABLE" == 1 ]] && tfrun python3 tensorflow/tools/ci_build/update_version.py --nightly
```

Comment:
For consistency this one needs broken into if statement.

Reply:
Thanks, I missed this one.